framework/wazuh/core/indexer/agent.py

```diff
@@ -66,17 +63,13 @@ async def create(
         Agent : dict
             The created agent instance.
         """
-        group_list = [DEFAULT_GROUP]
-        if groups is not None:
-            group_list.extend(groups)
-
         agent = Agent(
             id=id,
             name=name,
             raw_key=key,
             type=type,
             version=version,
-            groups=group_list,
+            groups=[DEFAULT_GROUP],
```

Comment:
No group should be specified when adding an agent to the index. The Wazuh Agent must make a request to the group-assigning endpoint so it is added both to the `agents` index and the configuration pulled from the `groups` index (to be developed in future issues of #27434 ).
```suggestion

```